### DISCUSSION TITLE
Adding initial configurations

### DISCUSSION THREAD

**Discussion by paullizer** [microsoft/LinuxBrokerForAVDAccess#3] [Announcements]
We are testing our Custom Script Extension that will auto configure all services for RHEL 8 to support connecting to brokered Linux desktops from AVD. We are also testing our Linux Session Release process, when a user disconnects from xRDP, this process initiates for making the system available. There is an intentional delay of 20 minutes where the user can return to AVD to receive the same Linux host they were using.

We will continue to release components as we move from dev and test into pre-release. This will require extension documentation along with bicep to simplify deployment and configuration.

**Comment by DavidHababou** [comment]
Hi Paul,
Do you recommend using this solution for production or its still under work?
